### DISCUSSION TITLE
rofi sponser or donate option

### DISCUSSION THREAD

**Discussion by RayZ0rr** [davatorium/rofi#1602] [Q&A]
Title. This is an essential piece of software for me. Anyway to fund or donate to it?

**Comment by DaveDavenport** [comment]
I don't have anything setup for donations currently.

**Reply by RayZ0rr** [reply]
I was checking through some of my favourite projects. Nvim has [github sponsors](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-user-account) and there's also [buymecoffee](https://www.buymeacoffee.com/junegunn) and [paypal](https://www.paypal.com/paypalme/junegunn) for fzf.
